images/dind/docker.service

```diff
@@ -1,15 +0,0 @@
-[Unit]
-Description=Docker-in-Docker (dind)
-After=network.target
```

Comment:
don't we need this dependency?
I was thinking if docker is started before network is up, some of the containers depend on network will fail.